rust/automerge-wasm/src/lib.rs

```diff
@@ -580,15 +580,16 @@ impl Automerge {
     pub fn apply_patches(
         &mut self,
         object: JsValue,
+        return_patches: JsValue,
```

Comment:
I would have made a new method `getAndApplyPatches()` that returns the object value and removed the callback from the other method instead of adding a Boolean argument and complicating the return value for the simple case.